lib/trento/clusters/cluster_enrichment_data.ex

```diff
@@ -13,6 +13,7 @@ defmodule Trento.Clusters.ClusterEnrichmentData do
   @primary_key {:cluster_id, :binary_id, autogenerate: false}
   schema "clusters_enrichment_data" do
     field :cib_last_written, :string
+    field :nodes_attributes, :map, default: %{}
```

Comment:
do you mean just storing the whole cluster details?
It would be simple for the final merging, however the question then is: does it even make sense projecting the details then? 🤔

Reply:
For the record, currently what gets store is something like this
```
{
  "vmhdbqas01": {
    "lpa_hdq_lpt": "1643125033"
  },
  "vmhdbqas02": {
    "lpa_hdq_lpt": "30"
  }
}
```
which is basically node_name => attributes_enriching_the_attributes_in_the_read_model

Reply:
Not the whole details data, just the same data "struct" format. Like details being:
`%{nodes: [%{attributes: %{"lpa_hdq_lpt" => "1643125033"}}, %{attributes: %{"lpa_hdq_lpt" => "30}}]}`, so it resembles a partial details struct. Something that you could just maybe merge with some basic `Map.merge` usage (maybe combining with some other Enum functions)

Reply:
as you suggested I kept the same structure. Could indeed be helpful if we need to do something similar for other parts of the detail.